### ISSUE TITLE
Models with Space in the name

### ISSUE THREAD

**Issue by jdtsmith** [sy1vain/node-pjlink#10]
Great library, thanks.  Using it for a homebridge plugin.  One issue I found was that the default response parsing is to split responses on the space character.  My projector returns spaces in its reported model name (`INF2`), which is then truncated to the first word.  I fixed this by overriding `formatResult`:

```
Command.ModelCommand.prototype.formatResult = function(args) {
    return args.join(" ");
}
```

**Comment by sy1vain** [comment]
Nice, if you make a pull request I will merge it.

This library is not under active development anymore since I am not in possession of a PJLink device at the moment.

**Comment by sy1vain** [comment]
Merged pull request #11
